packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1433,23 +1433,23 @@ abstract class FlutterCommand extends Command<void> {
       description,
       '',
       'Global options:',
-      runner.argParser.usage,
+      '${runner?.argParser.usage}',
       '',
       usageWithoutDescription,
     ].join('\n');
     return help;
   }
 
-  ApplicationPackageFactory applicationPackages;
+  ApplicationPackageFactory? applicationPackages;
 
   /// Gets the parsed command-line option named [name] as `bool`.
-  bool boolArg(String name) => argResults[name] as bool;
+  bool boolArg(String name) => argResults?[name] as bool? ?? false;
```

Comment:
Should we be defaulting to `false` rather than returning `null`?

Reply:
I think we want `false`, it's checked as a bool everywhere:
https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/lib/src/commands/build_appbundle.dart#L89-L93
https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/lib/src/commands/config.dart#L104
https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/lib/src/commands/ide_config.dart#L161

Set here:
https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/lib/src/commands/run.dart#L158
Checked as bool here:
https://github.com/flutter/flutter/blob/9e88fe328efcf42d3037a965b39269f0805cd4ce/packages/flutter_tools/lib/src/commands/run.dart#L448
etc

Reply:
makes sense